packages/tools/pie-icons-react/rollup.config.mjs

```diff
@@ -17,7 +16,6 @@ export default [
             preserveModules: true,
         },
         plugins: [
-            del({ targets: ['esm/*'] }),
```

Comment:
These are already being deleted by the `clean` script so we don't need to do it again